<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ekran</class>
 <widget class="QWidget" name="Ekran">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="slider1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>630</y>
     <width>300</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="cleanButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>630</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Czyść</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetSliderButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>630</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Resetuj Slider</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>630</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Wyjście</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>580</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Wartość n = </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="lineButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>570</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Linia</string>
   </property>
  </widget>
  <widget class="QPushButton" name="circleButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>570</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Okrąg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="elipseButton">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>570</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Elipsa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="randomColorButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>520</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Losuj kolor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="drawButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>570</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Swobodne rysowanie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="colorRedButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>520</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelColors">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>510</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Paleta kolorów:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="colorGreenButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>520</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="colorBlueButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>520</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="colorWhiteButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>520</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="colorPurpleButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>520</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="colorYellowButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>520</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="colorOrangeButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>520</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bezierCurveButton">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>570</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Krzywa Beziera</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bsplineCurveButton">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>630</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Krzywa BSpline</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
